homeassistant/components/light/hue.py

```diff
@@ -130,9 +134,18 @@ def update_lights():
 
         for light_id, info in api_states.items():
             if light_id not in lights:
-                lights[light_id] = HueLight(int(light_id), info,
-                                            bridge, update_lights,
-                                            bridge_type=bridge_type)
+
+                if _CONFIGURING['ignore_unreachable']:
+                    lights[light_id] = HueLight(int(light_id), info,
```

Comment:
I put the `if` statement test with `_CONFIGURING` since config.get wasn't working inside the `update_light` method, and kept it out of the HueLight constructor for clarity. I see what you mean and I'll move it in with the other arguments.